src/constants/colors.js

```diff
@@ -11,6 +12,13 @@ export const dialogApplyButtonBgColor = "rgb(51, 51, 51)";
 export const dialogApplyButtonTextColor = "#FFFFFF";
 export const dialogHeaderDividerColor = "#F2F2F2";
 
+// TAB BAR
+
+export const tabBarBgColor = "rgb(243, 243, 243)";
+export const tabBarLabelColor = "rgb(88, 88, 88)";
+export const tabBarActiveLabelColor = "rgb(45, 47, 127)";
+export const tabBarShadowColor = "rgba(0, 0, 0, 0.1)";
```

Comment:
The shadow color specified in Zeplin is `rgba(0, 0, 0, 0.2)`. This results in a darker shadow, though. 10% opacity seems to match it more closely. Not sure what causes this. 🤔